src/windows/README

```diff
@@ -29,8 +29,8 @@ To build Kerberos 5 on Windows, you will need the following:
 * A version of Perl.  We recommend Strawberry Perl as it will work
   to build OpenSSL.
 
-* Some common Unix utilities such as sed/awk/cp/cat installed in the
-  command-line path.
+* Some common Unix utilities such as sed/awk/cp/cat/mv/rm installed in
```

Comment:
Grammatically, "such as" implies that the list is incomplete and may just contain examples of the sort of thing being described.  If we believe that the list of utilities is now a complete list, we might want to remove the "such as" phrasing and change the punctuation accordingly, e.g., "Some common Unix utilities (sed/awk/cp/cat/mv/rm)  installed in the command-line path". 
That said, I do not object to the change as-is.